package.json

```diff
@@ -9,6 +9,7 @@
     "test": "lerna run test --stream --concurrency=1 -- --colors",
     "ci": "./scripts/ci.sh",
     "geth": "./scripts/geth.sh",
+    "dependency-graph": "lerna-dependency-graph -f png -o dependency-graph.png",
```

Comment:
Would PDF be better? PNG looks so raster

Reply:
I'll have a look, but I thought PDF would just encapsulate the raster. If that's the case, I'll also check for a SVG output, as that might be a bit nicer.

Reply:
Ah, I see what you mean now - PDF is indeed better, as the text is text, so cmd-f works, etc... wilco